src/components/AnimatedStep.js

```diff
@@ -21,13 +21,24 @@ const defaultProps = {
 };
 
 const AnimatedStep = (props) => {
+    function makeSlideInTranslation(translationType, fromValue) {
```

Comment:
Can we move this to `styles.js` and import it from there? You can checkout `growlNotificationTranslateY` as an example for the implementation.

Reply:
Updated